Jenkinsfile

```diff
@@ -0,0 +1,86 @@
+pipeline {
+    agent none
+    environment {
+        RUST_VERSION='1.50.0'
+        RUST_NIGHTLY='nightly-2021-02-21'
+    }
+    options{
+        lock('singleton-build') 
```

Comment:
so this ensures only one benchmark will run at a time?

Reply:
yes